### ISSUE TITLE
validation precision of pytorch-ECO-Lite on Something-something V1

### ISSUE THREAD

**Issue by ntuyt** [mzolfaghari/ECO-efficient-video-understanding#39]
Hi, using the pytorch code provided.
I only obtain 38.3 accuracy on Something-something V1 validation set.
Which is worse than reported, any suggestions?

**Comment by ntuyt** [comment]
By the way, I use 16 segments settings.

**Comment by pencilzhang** [comment]
> By the way, I use 16 segments settings.

I am also using the pytorch code with 16 segments and obtain 39.3% top-1 accuracy on sth-sth v1 val set. 
Did you figure it out?

**Comment by ntuyt** [comment]
No, I have not figured out. It is unsolved.

**Comment by mzolfaghari** [comment]
@ntuyt @pencilzhang Unfortunately PyTorch version does not give exact accuracy we obtained by Caffe. We were able to reproduce similar performance on other datasets but for the Sth-sth_v1 the result is lower. This is is due to some minor difference between the two implementations we found later.

Please don't hesitate to reopen the issue if you had further questions.
